viewing_party/party.py

```diff
@@ -1,23 +1,223 @@
 # ------------- WAVE 1 --------------------
 
 def create_movie(title, genre, rating):
-    pass
+    if title and genre and rating:
+        return {
+            "title": title, 
+            "genre": genre, 
+            "rating": rating
+            }
+    return None
+
+def add_to_watched(user_data, movie):
+    if "watched" in user_data:
+        user_data["watched"].append(movie)
+    else:
+        user_data["watched"] = [movie]
+
+    return user_data

```

Comment:
The requirements say that the dictionary will have a key "watched" so we can remove the conditional checks here to simplify this function:

```suggestion
    user_data["watched"].append(movie)

    return user_data
```